### PR TITLE
remove non-concurrent fields depth and max_depth from kndClass. Close…

### PR DESCRIPTION
**Pull request by M0nsteR**
…: #38

### CODE REVIEW

File: include/knd_repo.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ struct kndRepo
     const char *locale;
     size_t locale_size;
 
-    struct glbOutput *out;
+    //struct glbOutput *out;
```
Comment (NekoNe) [inline include/knd_repo.h]: May be it should be deleted?

File: include/knd_task.h [hunk 2]
```diff
@@ -121,21 +122,24 @@ struct kndTask
     size_t state_lte;
     bool show_removed_objs;
     bool show_rels;
+
+    size_t depth;
```
Comment (NekoNe) [inline include/knd_task.h:126]: What does depth field mean?

File: src/knd_attr.gsl.c [hunk 3]
```diff
@@ -64,7 +64,9 @@ static int inner_item_export_GSL(struct kndAttrVar *parent_item,
     if (parent_item->name_size) {
         //err = out->write(out, "{_id", strlen("{_id")); RET_ERR();
         //err = out->writec(out, ' '); RET_ERR();
-        err = out->write(out, parent_item->name, parent_item->name_size);          RET_ERR();
+
+        //err = out->write(out, parent_item->name, parent_item->name_size);          RET_ERR();
```
Comment (NekoNe) [inline src/knd_attr.gsl.c]: Should we delete unused code?

File: src/knd_attr.json.c [hunk 4]
```diff
@@ -343,8 +339,8 @@ static int ref_item_export_JSON(struct kndAttrVar *item,
     assert(item->class != NULL);
 
     c = item->class;
-    c->depth = item->depth;
-    c->max_depth = item->max_depth;
+    //c->depth = item->depth;
```
Comment (NekoNe) [inline src/knd_attr.json.c]: Comments

**Review by NekoNe — CHANGES_REQUESTED** [review]

File: include/knd_repo.h (hunk 1 above)
Comment (M0nsteR) [reply include/knd_repo.h]: Sure. All previous dependencies on this field are removed.

File: include/knd_task.h (hunk 2 above)
Comment (M0nsteR) [reply include/knd_task.h:126]: The depth field in kndTask is used to track the level of graph expansion while exporting. Max_depth sets the upper limit and depth variable shows the current level. It needs to be documented somewhere.

File: src/knd_attr.json.c (hunk 4 above)
Comment (M0nsteR) [reply src/knd_attr.json.c]: Resolved.

**Review by NekoNe — APPROVED** [review]

**Pull request merged**